src/main/kotlin/app/utils/FileHelper.kt

```diff
@@ -0,0 +1,14 @@
+// Copyright 2017 Sourcerer Inc. All Rights Reserved.
+// Author: Anatoly Kislov (anatoly@sourcerer.io)
+
+package app.utils
+
+import java.nio.file.Path
+
+object FileHelper {
+    fun getFileExtension(path: Path): String {
+        val fileName = path.fileName.toString()
+        return fileName.substringAfterLast(
```

Comment:
For me too. I could add apache commons io for this simple function, but it looks like overkill.